compiler/quilt/tools/command.py

```diff
@@ -908,10 +895,7 @@ def install_via_requirements(requirements_str, force=False):
     else:
         yaml_data = yaml.load(requirements_str)
     for pkginfo in yaml_data['packages']:
-        owner, pkg, subpath, hash, version, tag = parse_package_extended(pkginfo)
-        package = owner + '/' + pkg
-        if subpath:
-            package += '/' + "/".join(subpath)
+        package, hash, version, tag = parse_package_extended(pkginfo)
```

Comment:
Since you're changing the line already, avoid using 'hash'?

Reply:
I am sorry, do you mean to prevent override built-in function?